src/popup/pages/Dashboard.vue

```diff
@@ -12,6 +12,18 @@
     <template #cards>
       <LatestTransactionsCard />
 
+      <DashboardCard
+        v-if="isAeAccount"
```

Comment:
I'm not 100% sure if we should remove this card on non AE accounts. Did you consult this with Tsvetan?